File: python/src/Behavioral/Visitor/AbstractTitleInfo.py
#//AbstractTitleInfo.java - the abstract Visitee

class AbstractTitleInfo(object):

	titleName = None

	def __init__(self,titleName):
		self.setTitleName(titleName)

	def setTitleName (self,titleNameIn) :
		self.titleName = titleNameIn

	def getTitleName (self) : 
		return self.titleName

	def accept (self,titleBlurbVisitor) :
		raise Exception("Unimplemented method, AbstractTitleInfo.accept")

File: ruby/src/Behavioral/Visitor/AbstractTitleInfo.rb
#//AbstractTitleInfo - the abstract Visitee

class AbstractTitleInfo

	attr_accessor :titleName

	def initialize(titleName)
		setTitleName(titleName)
	end

	def setTitleName (titleNameIn)
 		@titleName = titleNameIn
	end

	def getTitleName ()
 		return @titleName
	end

	def accept (titleBlurbVisitor)
 		raise Exception("Unimplemented method, AbstractTitleInfo.accept")
	end
end

